bundles/af-core/src/main/java/com/adobe/cq/forms/core/components/models/form/FormComponent.java

```diff
@@ -73,6 +73,18 @@ default Map<String, Object> getDorProperties() {
         return Collections.emptyMap();
     }
 
+    /**
+     * Returns getPath of the form field
+     *
+     * @return getPath of the field
+     * @since com.adobe.cq.forms.core.components.util 3.1.0
+     */
+    @JsonView(Views.Author.class)
```

Comment:
this will be taken care of @devgurjar . we are introducing a custom prop `fd:path`.

Reply:
added minimum requirement to unblock myself